File: python/Easy/Part003/Expand Alphabets.py
# Read the entire input line
S = input().strip()

result = ""  # To store the expanded string
count = 0

# Traverse through the input string
i = 0
while i < len(S):
    if S[i].isdigit():
        count = count * 10 + int(S[i])  # Handle multi-digit numbers
    else:
        result += S[i] * count  # Repeat the character count times
        count = 0  # Reset count for the next number
    i += 1

# Output the expanded string
print(result)

File: c++/Easy/Part003/Expand Alphabets.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string S;
    getline(cin, S);  // Read the entire input line
    
    string result;  // To store the expanded string
    int count = 0;
    
    // Traverse through the input string
    for (int i = 0; i < S.length(); ++i) {
        if (isdigit(S[i])) {
            count = count * 10 + (S[i] - '0'); // Handle multi-digit numbers
        } else {
            result += string(count, S[i]); // Repeat the character count times
            count = 0; // Reset count for the next number
        }
    }

    // Output the expanded string
    cout << result << endl;

    return 0;
}
